ui/apps/platform/cypress/integration/collections/deploymentMatching.test.js

```diff
@@ -86,13 +86,16 @@ describe('Collection deployment matching', () => {
         assertDeploymentsAreMatched(['calico-node']);
         assertDeploymentsAreNotMatched(['kube-dns']);
 
-        // View more and ensure the next page loads
-        cy.get(`${selectors.resultsPanel} > div:last-child`).scrollTo('bottom');
+        // target `kube-dns` deployment is on the second or third page of results, so load three pages
```

Comment:
Aha, thank you for explicit comment about assumptions and implementation.

Important if a test fails because the state of central or the environment is not as a test expects.